AnkiConnect.py

```diff
@@ -434,6 +434,62 @@ def guiAddCards(self):
         addCards.activateWindow()
 
 
+    def getNextCard(self):
+        if (self.window().reviewer.card is None) or (self.window().state != "review"):
+            self.window().moveToState('review')
+
+        card = self.window().reviewer.card
+        nextCard = {}
+        nextCard['id'] = card.id
+        nextCard['question'] = card._getQA()['q']
+        nextCard['answer'] = card._getQA()['a']
+        nextCard['answerButtons'] = self.window().reviewer._answerButtonList()
+        nextCard['model_name'] = card.note(reload)._model['name']
+        nextCard['ord'] = card.ord
+        nextCard['fields'] = json.dumps(card.note(reload).fields)
+        nextCard['_fmap'] = card.note(reload)._fmap
+        nextCard['success'] = 'true'
+        return json.dumps(nextCard)
+
+
+    def showQuestion(self):
+        if (self.window().reviewer.card is None) or (self.window().state != "review"):
+            self.window().moveToState('review')
+
+        self.window().reviewer._showQuestion()
+        return json.dumps({'success': 'true'})
```

Comment:
How about you just return `None` if there is no error, otherwise a string with the error description? By the way, you can just leave out the return statement. No return at end of function is the same as `return None`